app/services/sufia/admin_set_service.rb

```diff
@@ -0,0 +1,5 @@
+module Sufia
```

Comment:
Good catch! I've changed `app/views/curation_concerns/base/_form_relationships.html.erb` to reference this new class.